jax/_src/extend/ffi.py

```diff
@@ -100,6 +100,14 @@ def include_dir() -> str:
   return os.path.join(jaxlib_dir, "include")
 
 
+def _aval_shape(aval: core.AbstractValue) -> Shape:
+  return () if aval is core.abstract_token else aval.shape
```

Comment:
Can you add the pytype exception here:

```suggestion
  return () if aval is core.abstract_token else aval.shape  # pytype: disable=attribute-error
```

Without this, our internal linters are complaining. Sorry!

Reply:
Oh, we explicitly opt into this behavior for mypy: https://github.com/google/jax/blob/c1d3c2db9f4e63a99c479a663ace47540ee36054/pyproject.toml#L7
I.e., `attr-defined` seems to allow access to statically-unknown attributes and types them as `Any`.